### ISSUE TITLE
filter_ts with shorter training time

### ISSUE THREAD

**Issue by davharris** [weecology/bbs-forecasting#123]
Right now, we're filtering the sites so that we have at least 25 observations between 1982 and 2013. Does that need to change if we're only doing 10 years of training? This could give us some sites with only 4 years of training data.

Maybe the criterion should be more like "visited during at least 70% of the training years"? Then we could use the same criterion for both.

As far as I can tell, there's no downside to including a site that doesn't get visited much in the test set, right?

**Comment by ethanwhite** [comment]
Good point. The proposed solution sounds good to me.

**Comment by sdtaylor** [comment]
:+1:
